pkg/apis/nginx/v1alpha1/nginx_types.go

```diff
@@ -37,6 +38,19 @@ type NginxSpec struct {
 	// Resources requirements to be set on the NGINX container.
 	// +optional
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
+	// Cache allows configuring a cache volume for nginx to use.
+	// +optional
+	Cache NginxCacheSpec `json:"cache,omitempty"`
+}
+
+type NginxCacheSpec struct {
+	// InMemory if set to true creates a memory backed volume.
+	InMemory bool `json:"inMemory,omitempty"`
+	// Path is the mount path for the cache volume.
+	Path string `json:"path,omitempty"`
```

Comment:
I would drop the `omitempty` annotation since this field is required.

```suggestion
	Path string `json:"path"`
```

Reply:
Done